### PR TITLE
Update contracts from peg-fix

### LINKED ISSUE #89: Update contracts from peg-fix
Need to update:
 - Bridge & Peg contract addresses
 - Peg ABI
 - Add validators to withdraw proof

### PR DESCRIPTION
**Pull request by aidan-starke**
## Description

- closes #89

#### Changes

- BridgeAddress & ERC20PegAddress for both Mainnet & Kovan
- update ERC20Peg ABI
- update tests to reflect changes

#### Adds

- validators to withdraw proof

### CODE REVIEW

File: components/bridge/Withdraw.tsx [hunk 1]
```diff
@@ -166,6 +166,22 @@ const Withdraw: React.FC<{
...
+				if (
+					ethers.utils.hexlify(validator) ===
+					ethers.utils.hexlify(
+						"0x000000000000000000000000000000000000000000000000000000000000000000"
+					)
+				) {
+					return ethers.constants.AddressZero;
+				} else {
+					return ethers.utils.computeAddress(validator);
+				}
```
Comment (ken-centrality) [inline components/bridge/Withdraw.tsx:181]: don't need `else` here

File: providers/BridgeProvider.tsx [hunk 2]
```diff
@@ -46,12 +46,12 @@ const BridgeProvider: FC<{ ethChainId: string }> = ({
...
-						ERC20PegAddress = "0x8F68fe02884b2B05e056aF72E4F2D2313E9900eC";
+						BridgeAddress = "0xf7997B93437d5d2AC226f362EBF0573ce7a53930";
+						ERC20PegAddress = "0x76BAc85e1E82cd677faa2b3f00C4a2626C4c6E32";
 						break;
 					case "42":
-						BridgeAddress = "0x9AFe4E42d8ab681d402e8548Ee860635BaA952C5";
-						ERC20PegAddress = "0x5Ff2f9582FcA1e11d47e4e623BEf4594EB12b30d";
+						BridgeAddress = "0x6484A31Df401792c784cD93aAAb3E933B406DdB3";
+						ERC20PegAddress = "0xa39E871e6e24f2d1Dd6AdA830538aBBE7b30F78F";
 						break;
```
Comment (ken-centrality) [inline providers/BridgeProvider.tsx:55]: I assume these addresses are applicable on both `Nikau` and `Azalea`?

File: tests/bridge/withdraw.test.ts [hunk 3]
```diff
@@ -143,6 +143,21 @@ describe("withdrawEthSide", () => {
...
+				if (
+					ethers.utils.hexlify(validator) ===
+					ethers.utils.hexlify(
+						"0x000000000000000000000000000000000000000000000000000000000000000000"
+					)
+				) {
+					return ethers.constants.AddressZero;
+				} else {
+					return ethers.utils.computeAddress(validator);
+				}
```
Comment (ken-centrality) [inline tests/bridge/withdraw.test.ts:158]: Same, don't need `else` block here

**Review by ken-centrality — APPROVED** [review]
Couple of questions, otherwise it's good to merge!

File: providers/BridgeProvider.tsx (hunk 2 above)
Comment (aidan-starke) [reply providers/BridgeProvider.tsx:55]: yup both contracts have been updated on both networks

**Review by eupston — APPROVED** [review]

**Pull request merged**